src/index.js

```diff
@@ -99,8 +99,9 @@ class Node extends EventEmitter {
     }
 
     // Attach remaining APIs
-    this.peerRouting = peerRouting(this)
-    this.contentRouting = contentRouting(this)
+    // If peer or content routing modules have been provided, use those, otherwise use the dht
+    this.peerRouting = this._modules.peerRouting || peerRouting(this)
+    this.contentRouting = this._modules.contentRouting || contentRouting(this)
```

Comment:
@diasdavid my original thought was to either use the provided peer and content routing modules, or the dht. After finishing up the core of the work and thinking more about the example and usage I feel like it would be better to use both if they are enabled. The thought being that leveraging a delegate is a good way to bootstrap a node into the network while it grows its own network, but that overtime its network would have a lot of the information. We could also change peer and content routing to allow for multiple modules and in the future potentially introduce some priority logic for ordering the calls to prevent flooding the network, if that becomes an issue.

Reply:
Think of it as a Stream Muxer or a Transport, you can use one, many, all the same time, disable some, you name it :)